src/main/java/nl/tudelft/ti2206/group9/level/TrackParser.java

```diff
@@ -0,0 +1,131 @@
+package nl.tudelft.ti2206.group9.level;
+
+import nl.tudelft.ti2206.group9.entities.AbstractEntity;
+import nl.tudelft.ti2206.group9.entities.Coin;
+import nl.tudelft.ti2206.group9.entities.Fence;
+import nl.tudelft.ti2206.group9.entities.Log;
+import nl.tudelft.ti2206.group9.entities.Pillar;
+import nl.tudelft.ti2206.group9.util.Point3D;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Mathias
+ */
+public class TrackParser {
+
+    /** Standard Y-value of the center of a Fence. */
+    public static final double FENCE_CENTER_HEIGHT = 2.8;
+
+    /**
+     * Parse all txt-files into TrackParts.
+     * @return List of TrackParts
+     *
+     */
+    public final List<TrackPart> parseTrack() {
+        List<TrackPart> partList = new ArrayList<TrackPart>();
+
+        File folder = new File("src/main/resources/nl/tudelft/ti2206/group9/level");
+        for (File file : folder.listFiles()) {
+            try {
+                TrackPart part = parseTrackPart(file.getPath());
+                partList.add(part);
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
Shouldn't the catch do more than just display the stack trace? If you want to keep your program running in case of an exception the trace is not going to help the user.

However our program is not likely to raise this exception, so leave it for now.

< review